server/gateway/src/routes/loaderFramed.ts

```diff
@@ -64,12 +63,6 @@ export function create(
                 winston.info(`Redirecting to ${redirectUrl}`);
                 response.redirect(redirectUrl);
             } else {
-                const jwtToken = jwt.sign(
-                    {
-                        user: request.user,
-                    },
-                    jwtKey);
-
```

Comment:
This is not the same token. And you can definitely use jwt for it because its on the server. So dont change it